packages/shared-ux/chrome/navigation/src/ui/components/navigation_section_ui.tsx

```diff
@@ -65,14 +66,13 @@ const navigationNodeToEuiItem = (
           }
         : undefined,
     href,
-    renderItem: getRenderItem(),
+    // renderItem: getRenderItem(),
```

Comment:
This functionality allowed teams to render any content inside the navigation node for maximum flexibility. Do we plan to keep allowing it?

Reply:
I believe we must keep it, since that plan has not changed. I think support for this must be carried into `EuiCollapsibleNavSubItemProps` as commented above.

Reply:
@tsullivan Can you clarify on this statement? Are you saying you need me to update `EuiCollapsibleNavItem` on the EUI side of things with a new `renderItem` prop?

Or alternatively, if this is meant to render custom top-level items, could you make this a conditional replacement on Kibana's side, e.g.

```tsx
{getRenderItem ? getRenderItem() : <EuiCollapsibleNav {...props} />}
````

Reply:
I need to sync with Seb on this, will get back to you next week.